pillar/edx/ansible_vars/residential.sls

```diff
@@ -107,52 +64,6 @@ edx:
       location: /edx/app/nginx/gitreload.htpasswd
 
   ansible_vars:
-    ### EDXAPP ENVIRONMENT ###
-    EDXAPP_MEMCACHE:
-      {% for cache_config in cache_configs %}
-      {% set cache_purpose = cache_config.get('purpose', 'shared') %}
-      {% if cache_purpose in purpose %}
-      {% set ELASTICACHE_CONFIG = salt.boto3_elasticache.describe_cache_clusters(cache_config.cluster_id[:20].strip('-'), ShowCacheNodeInfo=True)[0] %}
-      {% for host in ELASTICACHE_CONFIG.CacheNodes %}
-      - {{ host.Endpoint.Address }}:{{ host.Endpoint.Port }}
-      {% endfor %}
-      {% endif %}
-      {% endfor %}
-    ### XQUEUE ENVIRONMENT ###
-    XQUEUE_QUEUES:
-        'MITx-42.01x': 'https://xserver.mitx.mit.edu/fgxserver'
-        'MITx-8371': 'https://xqueue.mitx.mit.edu/qis_xserver'
-        # TODO: Are these courses still in use? Can we shut down the xserver instance? (tmacey 2017-03-16)
-        'MITx-6.s064x': 'http://127.0.0.1:8051'
-        'MITx-7.QBWr': 'http://127.0.0.1:8050'
-        'matlab': 'https://mitx.mss-mathworks.com/stateless/mooc/MITx'
-        # push queue
-        'edX-DemoX': 'http://localhost:8050'
-        # pull queues
-        'Watcher-MITx-6.0001r': !!null
-        'Watcher-MITx-6.00x': !!null
-        'open-ended': !!null
-        'open-ended-message': !!null
-        'test-pull': !!null
-        'certificates': !!null
-    XQUEUE_LOGGING_ENV: {{ edxapp_log_env_suffix }}
-    XQUEUE_DJANGO_USERS:
-      {{ edxapp_xqueue_creds.data.username }}: {{ edxapp_xqueue_creds.data.password }}
-      {{ xqwatcher_xqueue_creds.data.username }}: {{ xqwatcher_xqueue_creds.data.password }}
-    XQUEUE_AWS_ACCESS_KEY_ID: __vault__:cache:aws-mitx/creds/mitx-s3-{{ purpose }}-{{ environment }}>data>access_key
-    XQUEUE_AWS_SECRET_ACCESS_KEY: __vault__:cache:aws-mitx/creds/mitx-s3-{{ purpose }}-{{ environment }}>data>secret_key
-    XQUEUE_BASIC_AUTH_USER: mitx
-    XQUEUE_BASIC_AUTH_PASSWORD: __vault__::secret-residential/global/xqueue-password>data>value
-    XQUEUE_MYSQL_DB_NAME: xqueue_{{ purpose_suffix }}
-    XQUEUE_MYSQL_HOST: {{ MYSQL_HOST }}
-    XQUEUE_MYSQL_PASSWORD: __vault__:cache:mysql-{{ environment }}/creds/xqueue-{{ purpose }}>data>password
-    XQUEUE_MYSQL_PORT: {{ MYSQL_PORT }}
-    XQUEUE_MYSQL_USER: __vault__:cache:mysql-{{ environment }}/creds/xqueue-{{ purpose }}>data>username
-    XQUEUE_UPLOAD_BUCKET: mitx-grades-{{ purpose }}-{{ environment }}
-    xqueue_source_repo: {{ purpose_data.versions.xqueue_source_repo }}
-    xqueue_version: {{ purpose_data.versions.xqueue }}
-    ########## END XQUEUE ########################################
-
     ########## START THEMING ########################################
     EDXAPP_ENABLE_COMPREHENSIVE_THEMING: true
     EDXAPP_COMPREHENSIVE_THEME_SOURCE_REPO: '{{ purpose_data.versions.theme_source_repo }}'
```

Comment:
Doesn't look like `purpose_data` is any longer defined

Reply:
It should be...